cmds/core/watchdog/watchdog_linux_test.go

```diff
@@ -0,0 +1,154 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"os"
+	"testing"
+)
+
+// TestRun test watchdog cli against a regular file, most tests expected to
+// return an error, except of "keepalive"
+func TestRun(t *testing.T) {
+	dev, err := os.CreateTemp("", "")
```

Comment:
could you use t.TempDir() here? Then you would not need that defer below, etc.

Reply:
done